activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+*   Relation#destroy_all is performed in batches to prevent memory hogs.
+
+    This changes the documented type of Relation#destroy_all. If you depend on
+    it, set the following in your app:
```

Comment:
Actually, we should do the inverse. We default to `config.active_record.destroy_all_in_batches = false` and then we add an initializer for new Rails apps in 5.0 where we set `config.active_record.destroy_all_in_batches = true`. That way things will Just Work in both cases.